shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx

```diff
@@ -73,68 +73,56 @@ To implement this user interface, in your Unity project:
 
 Import the necessary .NET libraries, set up your <Vpl k="CLIENT" /> to run on Android, and request permissions for the camera and microphone.
 
-1. **Create a new script to hold the system logic**
+1. **Create a new script and import the required Unity libraray**
```

Comment:
It is a unity library, in that, it works in unity, However, you are importing an Agora SDK

Please change to an import <Vg name="whatever the tag is for video sdk" /> please.

Reply:
I did not import Agora SDK. I just explained how to create a script file and which library to import:

1. In **Project**, open **Assets** > **Agora-RTC-Plugin** > **Agora-Unity-RTC-SDK** > **Code**. Right-click **Code**, then click **Create** > **C# Script**. In **Assets** you see the `NewBehaviourScript` file that you use to implement <Vpd k = "PRODUCT"/> in your <Vpl k = "CLIENT"/>.

    1. In **Inspector**, click **Open**. `NewBehaviourScript.cs` opens in your default text editor.

    1. In `NewBehaviourScript.cs`, add the following namespace to the list of namespace declaration:

        ``` csharp
        using UnityEngine.UI;
        ```